dali/ft/ftbase.ipp

```diff
@@ -132,7 +132,7 @@ public:
 
     unsigned getCRC()               { return crc; }
     void setCRC(unsigned long _crc)     { crc = _crc; }
-
+    virtual unsigned __int64 getStatistic(StatisticKind kind) { return stream->getStatistic(kind); }
```

Comment:
style: Cleaner if this is moved next to the other virtuals.  Should also consider using override on this (and the other virtuals in this class, and elsewhere).   ( I think clang may complain if override is used on some virtuals in a class but not others...)